gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -90,21 +93,23 @@ public Consumer<Flux<String>> consumeTaskEventUpdate() {
     }
 
     void handleTaskEventUpdate(String loggerEventString) {
-        try {
-            ObjectMapper objectMapper = new ObjectMapper();
-            TaskLogEventUpdate loggerEvent = objectMapper.readValue(loggerEventString, TaskLogEventUpdate.class);
-            Optional<Task> optionalTask = taskRepository.findByIdWithProcessFiles(UUID.fromString(loggerEvent.getId()));
-            if (optionalTask.isPresent()) {
-                Task task = optionalTask.get();
-                OffsetDateTime offsetDateTime = OffsetDateTime.parse(loggerEvent.getTimestamp());
-                task.addProcessEvent(offsetDateTime, loggerEvent.getLevel(), loggerEvent.getMessage());
-                taskRepository.save(task);
-                taskUpdateNotifier.notify(task, false);
-            } else {
-                LOGGER.warn("Task {} does not exist. Impossible to update task with log event", loggerEvent.getId());
+        synchronized (LOCK) {
+            try {
+                ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Not related to this PR but: considering the number/frequency of events, i thnk it would be better if we create the objectMapper in the class level (as it is in a service it will be a singleton)...